js/atb.js

```diff
@@ -1,5 +1,5 @@
 var ATB = (() => {
-    var ddgRegex = '/duckduckgo\.com';
+    var ddgRegex = 'https:\/\/duckduckgo\.com/\?.*';
```

Comment:
- we want subdomains right?
- in quotes you don't need to escape the `/` (but the last one wasn't quoted, anyway)

maybe

```
var ddgRegex = 'https://(\W\.)?duckduckgo\.com/\?.*'
```